skimage/filters/thresholding.py

```diff
@@ -1,10 +1,15 @@
+import math
```

Comment:
```
In [2]: %timeit a = math.sqrt(100.)
The slowest run took 59.41 times longer than the fastest. This could mean that an intermediate result is being cached 
10000000 loops, best of 3: 68.2 ns per loop

In [3]: %timeit a = np.sqrt(100.)
The slowest run took 27.52 times longer than the fastest. This could mean that an intermediate result is being cached 
1000000 loops, best of 3: 511 ns per loop
```